bedrock/mozorg/templates/mozorg/about/manifesto.html

```diff
@@ -118,7 +118,7 @@ <h2 class="addendum-subtitle">{{ _('Pledge for a Healthy Internet') }}</h2>
       <div class="share-addendum">
         <h2 class="share-head">{{ _('Show Your Support') }}</h2>
         <p class="share-paragraph">{{ _('An internet with these qualities will not come to life on its own. Individuals and organizations must embed these aspirations into internet technology and into the human experience with the internet. The Mozilla Manifesto and Addendum represent Mozilla’s commitment to advancing these aspirations. We aim to work together with people and organizations everywhere who share these goals to make the internet an even better place for everyone.') }}</p>
-        <p class="share-button-outter"><a class="button button-manifesto js-manifesto-share" href="{{ twitter_link('custom', _('I support the vision of a better, healthier internet from @mozilla, will you join me?')) }}">{{  _('Share on Twitter') }}</a></p>
+        <p class="share-button-outter"><a class="button button-manifesto js-manifesto-share" target="_blank" href="{{ twitter_link('custom', _('I support the vision of a better, healthier internet from @mozilla, will you join me?')) }}">{{  _('Share on Twitter') }}</a></p>
```

Comment:
We should add `rel="noopener noreferrer"` to these links, for security and perf [benefits](https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/).